tests/gcloud-testing-core/src/main/java/com/google/cloud/runtime/jetty/test/AbstractIntegrationTest.java

```diff
@@ -128,8 +128,10 @@ public String getMode() {
   @BeforeClass
   public static void waitForServerUp() throws Exception {
     HttpUrlUtil.waitForServerUp(SERVER_URI, TIMEOUT_SECONDS, TimeUnit.SECONDS);
-    // wait for HTTPS as well
-    HttpUrlUtil.waitForServerUp(new URI(SERVER_URI.toASCIIString().replace("http:", "https:")),
-            TIMEOUT_SECONDS, TimeUnit.SECONDS);
+    // wait for HTTPS as well, but only for remote
+    if (!SERVER_URI.toASCIIString().startsWith("http://localhost")) {
```

Comment:
how about https://localhost? maybe `..contains("localhost")`?

Reply:
It will always start with `http://localhost` based on the code above. If you look at the body of the `if`, we're actually replacing `http` with `https`.
I thought of just doing a `contains` but that is error-prone itself.

Reply:
sounds good, thank you!